### PR TITLE
Update npm packages

### PR DESCRIPTION
**Pull request by danielailie**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,15 +39,15 @@
...
-    "@multiversx/sdk-core": "12.14.0",
-    "@multiversx/sdk-native-auth-server": "1.0.7",
-    "@multiversx/sdk-nestjs-cache": "3.4.1",
-    "@multiversx/sdk-nestjs-common": "3.4.1",
-    "@multiversx/sdk-nestjs-elastic": "3.4.1",
-    "@multiversx/sdk-nestjs-http": "3.4.1",
-    "@multiversx/sdk-nestjs-monitoring": "3.4.1",
-    "@multiversx/sdk-nestjs-redis": "3.4.1",
-    "@multiversx/sdk-network-providers": "^1.2.1",
+    "@multiversx/sdk-core": "13.4.2",
```
Comment (andreibancioiu) [inline package.json:42]: :rocket:

File: src/modules/proxy-deployer/tests/proxy-deployer.abi.service.spec.ts [hunk 2]
```diff
@@ -47,7 +47,7 @@ describe('Proxy  Deployer Abi Service', () => {
         sender: 'erd1dc3yzxxeq69wvf583gw0h67td226gu2ahpk3k50qdgzzym8npltq7ndgha',
         signature: undefined,
         value: '0',
-        version: 1,
+        version: 2,
```
Comment (andreibancioiu) [inline src/modules/proxy-deployer/tests/proxy-deployer.abi.service.spec.ts:50]: Just a double check - are there third party consumers of the NFT service that take crafted transactions and relay them? If so, if the construction of relayed V2 transactions on their side is done with old / obsolete components, they might encounter some issues (old / obsolete builders of relayed V2 do not know to use the `innerTx.version`).

File: src/common/services/mx-communication/mx-proxy.service.ts [hunk 3]
```diff
@@ -20,6 +20,7 @@ export class MxProxyService {
       timeout: parseInt(process.env.KEEPALIVE_TIMEOUT_DOWNSTREAM),
       httpAgent: mxConfig.keepAlive ? httpAgent : null,
       httpsAgent: mxConfig.keepAlive ? httpsAgent : null,
+      clientName: "nft-service"
```
Comment (andreibancioiu) [inline src/common/services/mx-communication/mx-proxy.service.ts:23]: :rocket:

**Review by andreibancioiu — APPROVED** [review]

File: src/modules/proxy-deployer/tests/proxy-deployer.abi.service.spec.ts (hunk 2 above)
Comment (danielailie) [reply src/modules/proxy-deployer/tests/proxy-deployer.abi.service.spec.ts:50]: no.. there are not, as far as I know

**Review by danidrasovean — APPROVED** [review]

**Pull request merged**